akd/src/ecvrf/no_vrf.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+//! This module contains mock usage of the VRF functionality should someone choose they
+//! don't want the overhead of VRF computation. It is not designed for testing, but to
+//! provide full functionality of the prod library but WITHOUT VRFs.
+
+use crate::{errors::VRFStorageError, node_state::NodeLabel, storage::types::AkdLabel};
+use async_trait::async_trait;
+use winter_crypto::Digest;
+use winter_crypto::Hasher;
+
+/// A mock VRF public key
+pub struct VRFPublicKey;
+
+impl VRFPublicKey {
+    /// This function is called to verify that a given NodeLabel is indeed
+    /// the VRF for a given version (fresh or stale) for a username.
+    /// Hence, it also takes as input the server's public key.
+    pub fn verify_label<H: Hasher>(
+        &self,
+        _uname: &AkdLabel,
+        _stale: bool,
+        _version: u64,
+        _proof: &[u8],
+        _label: NodeLabel,
+    ) -> Result<(), VRFStorageError> {
+        Ok(())
+    }
+}
+
+/// A mock VRF private key
+pub struct VRFPrivateKey;
+
+/// A mock VRF proof
+pub struct Proof;
+
+impl Proof {
+    /// Converts a Proof into bytes
+    pub fn to_bytes(&self) -> [u8; 80] {
+        [0u8; 80]
+    }
+}
+
+/// Represents a secure storage of the VRF private key. Since the VRF private key
+/// should change never (if it does, the entire tree is no longer a consistent mapping
```

Comment:
At some point we said we could embed this private key in the code. From this sentence, it sounds like that's something we shouldn't do. Maybe we should consider alternatives.

Reply:
We aren't in the end. I spoke to @kevinlewi about it async, and it's secret enough it should go into a secure storage medium. The "HardCodedVRFKey" is only used in testing, and someone should properly implement the trait to a secure storage layer for the private key.